lms/djangoapps/grades/api/v1/gradebook_views.py

```diff
@@ -1,22 +1,99 @@
 """
-Defines an endpoint for retrieving assignment type and subsection info for a course.
+Defines an endpoint gradebook related APIs for a course.
```

Comment:
NIT (for later PR): change to "Defines an endpoint for gradebook data related to a course."